### PR TITLE
directly send the icon url for activities to the qml component

### PR DESCRIPTION
**Pull request by mgallien**
should allow usage of default cache from Image qml standard component

Signed-off-by: Matthieu Gallien <matthieu.gallien@nextcloud.com>

### CODE REVIEW

File: src/gui/tray/ActivityListModel.cpp [hunk 1]
```diff
@@ -178,11 +178,7 @@ QVariant ActivityListModel::data(const QModelIndex &index, int role) const
             }
         } else {
             // We have an activity
-            if (!a._iconData.isEmpty()) {
-                const QString svgData = "data:image/svg+xml;utf8," + a._iconData;
-                return svgData;
-            }
-            return "qrc:///client/theme/black/activity.svg";
+            return a._icon;
```
Comment (nickvergessen) [inline src/gui/tray/ActivityListModel.cpp:181]: Just for the record, activity icons are optional (as per https://github.com/nextcloud/activity/blob/master/docs/endpoint-v2.md#activity-element )
Not sure if this is now unhandled or seomthing.

Comment (mgallien) [reply src/gui/tray/ActivityListModel.cpp:185]: thanks,
true, the original code had the check and I forgot to keep it

**Review by FlexW — APPROVED** [review]

**Review by allexzander — APPROVED** [review]

**Review by camilasan — APPROVED** [review]

**Comment by mgallien** [comment]
backport to stable-3.2

**Pull request merged**